composer.json

```diff
@@ -33,6 +33,9 @@
     "autoload": {
         "psr-4": {
             "Give\\": "src/"
-        }
+        },
+        "files": [
+            "src/Helpers/Helpers.php"
```

Comment:
You're kind of recreating the file autoloading in the `Helpers.php` file right now by not actually including any functions in it. Just forego the `Helpers.php` file and just list out the files you want auto-included here in the `composer.json` files array. That way all autoloading is clearly in one place.

Reply:
fixed: https://github.com/impress-org/givewp/compare/f60b190e8778...045cd0436d8c